datastore/common/objects/OperatingSystem.hpp

```diff
@@ -63,14 +64,17 @@ namespace netmeld::datastore::objects {
     private:
     protected:
     public:
+      void setIpAddr(const IpAddress&);
       void setVendorName(const std::string&);
       void setProductName(const std::string&);
       void setProductVersion(const std::string&);
       void setCpe(const std::string&);
       void setAccuracy(const double);
+      void setHotfixs(const std::vector<std::string>&);
+      void addHotfix(const std::string&);
 
       bool isValid() const override;
-
+      std::string ConvertVectorToPostgresArray(const std::vector<std::string>& vec);
```

Comment:
The idea was to have the hotfixes not be separate but to be stored in a postgres array. Instead of just a concatenated string. I can move it to utils if we will use it again?

Reply:
No, don't move it to utils.  It ultimately is unnecessary logic.

Regardless, I am marking this as resolved since it is outdated and will comment on the new code.